new items and fixes

Eddyskingdom opened the pull request:
shotgun small rework adding in also new ammo bexes and flags and emote sounds but still geting workt on

Harcourt1 commented:
Why did you remove those shells? I wont merge the shotgun rework but im fine with everything else. 
Beanbags dont kill and removing the ammo will break the guns.

Eddyskingdom commented:
> Why did you remove those shells? I wont merge the shotgun rework but im fine with everything else.
Beanbags dont kill and removing the ammo will break the guns <

what the hell @Harcourt1 have you even taken a look at the code i moved them to the casings code so spend shotgun shells will now appear and what do you mean beanbags wont kill they shoot a projectil so they must do damge and the shotguns are not change only how the shells work because first the projectile code was in the shotgun code but i moved it to the shells so shells shoot the projectile and not the gun when you shoot  it 

LOOK AT THE CHANGES THEN QUESTION MY CHANGES BECAUSE THIS MY 100 TIME I NEED TO TELL YOU THAT THE CODE IS FINE AND WORKS BECAUSE I TEST IT BEVOOR COMITING THE CHANGES INSTED OF YOU GOING TO RELEASE IT BEVOOR KNOWING WHAT YOU JUST DID!

Harcourt1 commented:
![image](https://user-images.githubusercontent.com/12753891/44313218-2be9c380-a3d2-11e8-8f4d-710b63963e49.png)

cariosecus commented:
Fix the conflicts and ill merge

Eddyskingdom commented:
ter moved into the _ammo_casing.dmi

Eddyskingdom commented:
Harcourt you a dick i was still working on that
